sns-sqs-fanout-cdk-java/example-pattern.json

```diff
@@ -0,0 +1,50 @@
+{
+	"title": "Fan-out pattern using SNS-SQS filters",
+	"description": "Creates SNS topic and Sqs queues with message body based and attribute based filter using configuration file (appSettings.json)",
+	"language": "Java",
+	"level": "200",
+	"framework": "CDK",
+	"introBox": {
+		"headline": "How it works",
+		"text": [
+			"This CDK application demonstrates how to create fan-out pattern using sns topic and sqs queues using subscription filters. ",
+			"Number of queues required and type of filter (Message attribute based or message body based) can be defined in appSettings.json (Sample json is provided, replace queue names and filter by conditions as needed)."
+		]
+	},
+	"gitHub": {
+		"template": {
+			"repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/sns-sqs-fanout-cdk-java",
+			"projectFolder": "sns-sqs-fanout-cdk-java",
+			"templateFile": "src/main/java/com/myorg/SnsToSqsFanOutStack.java.java",
+			"templateURL": "serverless-patterns/sns-sqs-fanout-cdk-java"
+		}
+	},
+	"resources": {
+		"bullets": [
```

Comment:
please include some relevant links, at least to the documentation about message filtering and SQS to SNS subscription process